core/src/trezor/ui/layouts/tt_v2/__init__.py

```diff
@@ -188,7 +188,7 @@ async def show_address(
     raise NotImplementedError
 
 
-def show_pubkey(
+async def show_pubkey(
```

Comment:
this looks wrong. the function is not async, it is returning an awaitable.

if this was right, caller would need to `await (await show_pubkey(...))`

(consider:
```python
async def int_later() -> int:
    await sleep(10)
    return 5

def int_wrapper() -> Awaitable[int]:
    awaitable_value = int_later()
    type(awaitable_value)  # Awaitable[int]
    return awaitable_value

async def int_user() -> None:
    # await function directly:
    first = await int_later()
    type(first)  # int
    # await the result of the call:
    awaitable = int_later()
    type(awaitable)  # Awaitable[int]
    second = await awaitable
    type(second)  # int
    # use the wrapper
    third = await int_wrapper()
    type(third)  # int
```
)

Reply:
Fixed in the force-push